release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
 An update for machine-config-daemon is now available for {product-title} 4.4.
 Details of the update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:2820[RHSA-2020:2820] advisory.
+
+[[ocp-4-4-12]]
+=== RHBA-2020:2871 - {product-title} 4.4.12 Bug Fix Update
+
+Issued: 2020-07-13
+
+{product-title} release 4.4.12 is now available. The list of container images and
+bug fixes included in the update are documented in the
+link:https://access.redhat.com/errata/RHBA-2020:2871[RHBA-2020:2871] advisory.
+The RPM packages included in the update are provided by the
+link:https://access.redhat.com/errata/RHBA-2020:2875[RHBA-2020:2875] advisory.
+
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5217641[{product-title} 4.4.12 container image list]
+
+[[ocp-4-4-12-bug-fixes]]
+==== Bug Fixes
+
+* Not blocking having several listeners, on different protocols, on the same
```

Comment:
I don't understand the first sentence. Perhaps what is meant is: "Having several listeners, on different protocols, on the same port for the `ovn-octavia` driver is blocked."

Reply:
I didn't quite understand either but I think this works: "Previously, having several listeners on different protocols on the same port for the `ovn-octavia` driver was not supported and was blocked. With this release, it is supported and there is no need to block it."